java8/src/main/java/com/dslplatform/json/processor/ConverterTemplate.java

```diff
@@ -109,10 +117,32 @@ private void asFormatConverter(final StructInfo si, final String name, final Str
 			code.append("\t\tprivate final java.lang.reflect.Type[] actualTypes;\n");
 		}
 
+		Map<String, TypeMirror> genericAttributes = new HashMap<>();
+		for (AttributeInfo attr : si.attributes.values()) {
+			if (!si.isParameterized && attr.isGeneric) {

```

Comment:
We can handle generics later.... but in that case at least leave a TODO comment

Reply:
We can lost the actual generic values later. I will double check this code.